charts/reform-scan-blob-router/values.preview.template.yaml

```diff
@@ -46,9 +46,9 @@ java:
     STORAGE_BLOB_PROCESSING_DELAY_IN_MINUTES: 0
 
     STORAGE_BULKSCAN_URL: https://bulkscan.aat.platform.hmcts.net
+    STORAGE_PCQ_URL: "https://pcqsharedaat.blob.core.windows.net"
```

Comment:
this does not seems correct to me.
is should be osba??

Reply:
we've been using aat storage account for bulkscan containers. wouldn't it be the same for pcq?